docs/advanced/public-api.md

```diff
@@ -0,0 +1,126 @@
+---
+id: public-api
+title: Public API
+
+---
+
+With Public API you can access ConfigCat platform programmatically. It is useful when you need to create, modify or
+delete any entities in ConfigCat. The API is based on HTTP REST, uses resource-oriented URLs, status codes and
+supports JSON and JSON+HAL format.
+
+## Authentication
+
+The API uses the ['Basic' HTTP Authentication Scheme](https://tools.ietf.org/html/rfc7617) where a
```

Comment:
we discussed about basic auth description and agreed that we dont explain the algorithm just mention and put a link.